plugins/org.jkiss.dbeaver.ext.postgresql/OSGI-INF/l10n/bundle_fr.properties

```diff
@@ -0,0 +1,347 @@
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndex.ready.description=Si true, l?index est pr\u00eat \u00e0 ajouter des donn\u00e9es. Une valeur false indique que l?index est ignor\u00e9 pour INSERTS/UPDATES.
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndex.unique.description=Index unique
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndex.valid.name=Valide
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndex.valid.description=Si true, l?index peut \u00eatre utilis\u00e9 dans les requ\u00eates. False signifie que l?index est peut-\u00eatre incomplet : il sera cependant modifi\u00e9 par les commandes INSERT/UPDATE, mais il est s\u00fbr de l?utiliser dans des requ\u00eates impossibles. Si elle est unique, la propri\u00e9t\u00e9 d?unicit\u00e9 n?est pas non plus garantie.
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndex.objectDefinitionText.description=D\u00e9finition de l'index
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndexColumn.ascending.name=Ascendant
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndexColumn.name.name=Nom
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndexColumn.ordinalPosition.name=Position ordinale
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreIndexColumn.tableColumn.name=Colonne de la table
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreLanguage.name.name=Nom
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreLanguage.objectId.name=ID de l'objet
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreLanguage.ownerId.name=Propri\u00e9taire ID
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreLanguage.trusted.name=Approuv\u00e9
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreLanguage.userDefined.name=D\u00e9fini par l'utilisateur
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreSetting.name.name=Nom
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreSetting.value.name=Valeur
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreProcedure.objectId.name=ID de l'objet
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreProcedure.aggregate.name=Regroup\u00e9
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreProcedure.body.name=Corps
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreProcedure.execCost.name=Co\u00fbt d'ex\u00e9cution
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreProcedure.language.name=Langage
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreProcedure.owner.name=Propri\u00e9taire
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreProcedure.owner.description=Le propri\u00e9taire de la fonction
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreProcedure.returnType.name=Type retourn\u00e9
+meta.org.jkiss.dbeaver.ext.postgresql.model.PostgreProcedure.strict.description=La fonction retourne NULL si l?un des arguments est NULL. Dans ce cas, la fonction ne sera pas appel\u00e9e du tout. Les fonctions qui ne sont pas "strictes" doivent \u00eatre pr\u00e9par\u00e9es pour accepter des valeurs NULL.
```

Comment:
It looks like `l?un` also has unexpected encoding.

Reply:
Fixed